minecraft_servers/fastmc/manifest.json

```diff
@@ -0,0 +1,29 @@
+{
+  "server_name": "fastmc",
+  "nice_name": "FastMC Network",
+  "direct_ip": "play.fastmc.net",
+  "server_wildcards": [
+    "%.fastmc.net"
+  ],
+  "supported_languages": [
+     "de"
+  ],
+  "social": {
+    "web": "https://www.fastmc.net",
+    "web_shop": "https://shop.fastmc.net",
```

Comment:
The shop is the same page as the normal page, isn't it?

Reply:
Yes, for now, but there will be an update of the store and the website where this is separated with subdomains.

Reply:
Then please remove the url until you have released the shop.